demisto_sdk/commands/common/constants.py

```diff
@@ -1615,6 +1621,10 @@ class PB_Status:
     FileType.PACK_IGNORE,
 )
 
+DEFAULT_MANDATORY_VALIDATIONS: Dict[str, Dict] = {
```

Comment:
Because we want to keep something that will make it as simple as possible for customers.
** DEFAULT_MANDATORY_VALIDATIONS** will hold all the mandatory validations and will be used if no file_path is given or found.. that way the customers will be able to use the new validator without doing anything in advance.